server/api/cvrs.py

```diff
@@ -68,7 +68,7 @@ def process():
 
         # Parse out all the initial metadata
         _election_name = next(cvrs)[0]
-        contest_row = next(cvrs)
+        contest_row = [" ".join(contest.splitlines()) for contest in next(cvrs)]
```

Comment:
the only suggestion i have is to pull this logic out into a utility function - maybe `util.remove_newlines` or something like that. that way it would be more clear what's happening here. but it's not blocking